src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -10,7 +10,7 @@
     <AssemblyName>System.Runtime.Tests</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>1718</NoWarn>
-    <NugetTargetMoniker Condition="'$(TargetGroup)'=='netstandard1.7' AND '$(NugetTargetMoniker)'==''">.NETStandard,Version=v1.7</NugetTargetMoniker>
+    <NugetTargetMoniker Condition="'$(TargetGroup)'=='netstandard1.7'">.NETStandard,Version=v1.7</NugetTargetMoniker>
```

Comment:
minor: This property is already set in dir.props (https://github.com/dotnet/corefx/blob/dev/api/dir.props#L363), based on the TargetGroup.